### PR TITLE
Adding pipelines scripts for setup & generation

### PR DESCRIPTION
**Pull request by AlonsoMondal**
Adding pipelines scripts for repo setup & SDK generation to remove dependencies to the private repository JenkinsSdkSetupScripts.

### CODE REVIEW

File: targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs [hunk 1]
```diff
@@ -103,6 +103,10 @@ public class PlayFabApiTest
         TitleData resultData = gson.fromJson(testTitleJson, new TypeToken<TitleData>(){}.getType());
         PlayFabSettings.TitleId = resultData.titleId;
         USER_EMAIL = resultData.userEmail;
+        String devSecretKey = env.get("DEVELOPER_SECRET_KEY_JAVA_SDK");
```
Comment (amccalib) [inline targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs:106]: Was this change intended to be included? I don't see how it's related to the Jenkins script move.

**Review by amccalib — APPROVED** [review]

File: targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs (hunk 1 above)
Comment (AlonsoMondal) [reply targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs:106]: Yes, JenkinsSdkSetupScripts contained the title data including the developer secret key. In order to remove the dependency to that repo & not have the secret published in a public repo, the pipelines set it as an env var. Suggestions are welcome

Comment (amccalib) [reply targets/java/source_shared/src/test/java/com/playfab/test/PlayFabApiTest.java.ejs:106]: Oh, I wasn't aware of that. Your solution is fine.
